pootle/templates/editor/units/edit.html

```diff
@@ -13,6 +13,13 @@
       </ul>
     </div>
     <div class="translate-{% locale_align %}">
+      <!-- Priority -->
```

Comment:
The biggest concern here is that this couples the editor and vfolders, and this is meant to be an optional component. While I'm okay to live with this for the time being, we need to exercise and elaborate on actual approaches to inject 3rd party code and behavior.